### DISCUSSION TITLE
Data Presentation 11/14/2024

### DISCUSSION THREAD

**Discussion by umarbilgrammi** [ecn310/course-project-taxes-tariffs#19] [General]
data presentation :)

**Comment by umarbilgrammi** [comment]
### **11/13/2024**

- Working Hypothesis: If Value added tax or sales tax revenue increases in a developing country, then tariff revenue decreases.
- Null Hypothesis: In developing countries, there is no difference in revenue when value added tax or tariffs change. 
- Data we used: https://github.com/ecn310/course-project-taxes-tariffs/commit/57ee616794a8ca4ddeaa2bdd0acabb8aefbe8238 | it is titled (merged.dta)
- Focus: pwcorr international vat --> -0.3250 
-  twoway (histogram vat, color(blue%50)) (histogram international, color(red%50)) --> 2 way histogram 
 - scatter international vat --> scatterplot
- This shows that there is a weak to moderate negative relationship between tariffs and value added tax in developing countries. Thus, we reject the null hypothesis. Preliminary data tests show that our working hypothesis may be true, however we need to conduct further data tests. What other tests should we conduct? 

<img width="large" alt="Screenshot 2024-11-13 at 5 42 42 PM" src="https://github.com/user-attachments/assets/c9b8e343-20b8-4d1e-ad3d-aa1374e66dbe">

we need to add X-axis as mean of % revenue 


<img width="large" alt="Screenshot 2024-11-13 at 5 44 06 PM" src="https://github.com/user-attachments/assets/dac551eb-b0e2-44ec-bb5b-ab14dd8baf58">

- Negative line can be drawn from top left to the right showing the negative trend 

- We were unable to make a table since there are too many values 

- Is there any other visual forms of data which would work? We want to add as much figures as possible.

**Comment by Ksarrge** [comment]
## 11/14/2024 Prof Buzard's Feedback:
- Make x-axis/y-axis clearer for histogram & scatterplot, need a key for histogram
- figure out what outlier in histogram is
  - time-series data: figure out outlier country and see their data throughout the years
- more countries have relatively small amount of revenue, what's the mean for each of these variables across distribution
- if a country has a zero value for tariffs/tax, should we keep it?
  - we need to tell the story around it and what people need to know to understand it, what countries have no or very small tariff revenue
- export subsidies = gov could be paying people to export
- could we time-series the whole dataset? = take subset of it and do line graphs to see trends or summary stack to see average/ average by year
- need more on the data, what is the data source/what's it covering/the focus/ time frame
- we have a negative correlation but in order to believe the rejection of our null hypothesis, we need to find the p-value: pwcore, sig
- 2 sample ttest for splitting it into developing vs developed countries and compare variables

Scatter Plot:
- add regression line
- add shading to data points or make them smaller
- take out countries with zero tariff value

**Comment by ryanpseely** [comment]
@ecn310/taxes-tariffs Great work today!

Here is one  graphics idea I forgot to mention in class: 

From my understanding, your group's theory on this says that "Value Added Taxes + Tariffs = Same Portion of Revenue," where if one of the two revenue sources goes up, the other will go down and leave the portion of revenue largely constant.
One way to show this concept would be to use a stacked bar chart. Each bar could represent the average of a number of countries or each bar could represent the average of each year in your data, you can play around with it and see what makes the most sense. You can also set it up where the y-axis represents the total amount of revenue (each bar would have a different height) or where the y-axis represents the proportion of revenue (percentage stacked bar chart) where each bar is the same height but then you don't see the differences in total amount of revenue.

Here is a link to an explainer on stacked bar charts and some options on how to present your data within one: https://www.atlassian.com/data/charts/stacked-bar-chart-complete-guide#:~:text=The%20stacked%20bar%20chart%20(aka,of%20the%20second%20categorical%20variable.
